### ISSUE TITLE
DB-Update for OpenWebAuth doesn't work

### ISSUE THREAD

**Issue by nupplaphil** [friendica/friendica#5278]
### Expected behavior

`bin/console dbstructure update` with no output (= worked)

### Actual behavior

`bin/console dbstructure update` with the following output:
```console
ALTER IGNORE TABLE `openwebauth-token` MODIFY `uid` int(10) unsigned NOT NULL DEFAULT 0 COMMENT 'User id', MODIFY `type` varchar(32) DEFAULT '' COMMENT 'Verify type', MODIFY `token` varchar(255) DEFAULT '' COMMENT 'A generated token', MODIFY `meta` varchar(255) DEFAULT '' COMMENT '';
```

### Steps to reproduce the problem

hard to say ..  it's a running environment and last time I did an update was about 1 week ago.
- git pull latest `develop`
- `bin/console dbstructure update`

But the node is running so no "blocker"

### Friendica version you encountered the problem

`2018.08-dev`

### Friendica source (git, zip)

git

### PHP version

```console
PHP 7.1.16 (cli) (built: Apr  5 2018 04:48:10) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.1.16, Copyright (c) 1999-2018, by Zend Technologies
```

### SQL version

```console
mysql  Ver 15.1 Distrib 10.3.7-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

**Comment by MrPetovan** [comment]
When a query shows up, it means it didn't run?

**Comment by nupplaphil** [comment]
To be honest, I don't know if it didn't run. But everytime I make the dbstructure update, this query appears (tried it 3-4 times).

afaik when a query means that this part isn't working during the db-update, is it?

**Comment by MrPetovan** [comment]
I have no idea, I'm utterly confused by this tool I though I knew how it worked.

**Comment by nupplaphil** [comment]
This is the current table structure of `openwebauth-token`:
```console
+---------+------------------+------+-----+---------------------+----------------+
| Field   | Type             | Null | Key | Default             | Extra          |
+---------+------------------+------+-----+---------------------+----------------+
| id      | int(10)          | NO   | PRI | NULL                | auto_increment |
| uid     | int(10) unsigned | NO   |     | 0                   |                |
| type    | varchar(32)      | YES  |     |                     |                |
| token   | varchar(255)     | YES  |     |                     |                |
| meta    | varchar(255)     | YES  |     |                     |                |
| created | datetime         | NO   |     | 0001-01-01 00:00:00 |                |
+---------+------------------+------+-----+---------------------+----------------+
```

**Comment by MrPetovan** [comment]
So the create went through, and now the computed table structure doesn't match our table structure array?

**Comment by nupplaphil** [comment]
I don't see any difference between the current table structure and what the `ALTER...` query tries to modify..

**Comment by nupplaphil** [comment]
OK one more strange thing: 
The `post_update_version` in the database is `1194`
Is that right? I think that has to be `1269` (the DB_UPDATE_VERSION) after the update, hasn't it?

**Comment by annando** [comment]
I'm currently having a look at it.

**Comment by nupplaphil** [comment]
Now it works :-) (= no more output)
thanks @annando for the #5279

**Comment by MrPetovan** [comment]
Caused by #5253, closed via #5279
